### DISCUSSION TITLE
How do you apply props to progressCircle in Picker?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [adobe/react-spectrum#5901] [Q&A]
I'm new to React Spectrum and really like the Picker component. However, I'm having a tough time understanding how to do simple things, such as modify styles. In particular, I'd like to change the color on the progressCircle that appears when loading async data into the list.  I see that `ProgressCircle` itself has props to change the color to white or black, but I can't figure out how to access these props within Picker (other colors, I have no idea).  Any suggestions?

Thanks

**Comment by reidbarber** [comment]
Hello! Custom styling options in React Spectrum are intentionally limited in order to ensure that usage is in line with the Spectrum design system. You can read more about that in the [Styling](https://react-spectrum.adobe.com/react-spectrum/styling.html) section of the React Spectrum docs.

For that specific scenario, the `staticColor` prop on ProgressCircle is meant to be used when it is over a background color or image ([Spectrum guidelines](https://spectrum.adobe.com/page/progress-circle/#Over-background-variant)). This wouldn't be applicable inside a picker, since the background is the Picker's popover.

For cases where you want to use custom styling, you can use the [Select](https://react-spectrum.adobe.com/react-aria/Select.html) and [ProgressBar](https://react-spectrum.adobe.com/react-aria/ProgressBar.html#circular) from React Aria, where there are lots of different ways to do [custom styling](https://react-spectrum.adobe.com/react-aria/styling.html).

**Comment by dungle-scrubs** [comment]
@reidbarber thanks, I understand now. I wonder if it's working properly for me though, because I'm seeing this ProgressCircle not in the popover, but the main element itself, while the initial list items are being fetched. 

![CleanShot 2024-02-19 at 08 48 43](https://github.com/adobe/react-spectrum/assets/13479539/740b30c8-20f1-4c0d-85b9-b93ab31ce5c9)

Either way, no worries and thank you for the links. I really like the way the Picker functions on mobile devices by the way.

**Reply by reidbarber** [reply]
No problem! There is also a ProgressCircle used inside the popover (shown in the[ Async loading example](https://react-spectrum.adobe.com/react-spectrum/Picker.html#asynchronous-loading) when you open the picker and scroll down to load more items). That was the one I thought you were referring to.
